src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/TestsResources.xaml

```diff
@@ -622,4 +623,334 @@
 			</Setter.Value>
 		</Setter>
 	</Style>
+
+	<!-- ComboBox -->
+
+	<Style x:Key="ComboBoxWithSeparatorStyle"
+			   TargetType="ComboBox"
+			   BasedOn="{StaticResource DefaultComboBoxStyle}">
+		<Setter Property="BorderBrush"
+					Value="{StaticResource SeparatorBrush}" />
+		<Setter Property="BorderThickness"
+					Value="0,0,0,1" />
+		<Setter Property="Margin"
+					Value="60,0" />
+	</Style>
+
+	<Style x:Key="DefaultComboBoxItemStyle"
+			   TargetType="ComboBoxItem">
+		<Setter Property="Background"
+					Value="Transparent" />
+		<Setter Property="VerticalContentAlignment"
+					Value="Center" />
+		<Setter Property="HorizontalContentAlignment"
+					Value="Stretch" />
+		<Setter Property="Padding"
+					Value="0" />
+		<Setter Property="Height"
+					Value="48" />
+
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="ComboBoxItem">
+					<Grid x:Name="RootGrid"
+							  Background="{TemplateBinding Background}">
+
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="CommonStates">
+								<VisualState x:Name="Normal" />
+
+								<VisualState x:Name="PointerOver">
+									<Storyboard>
+										<DoubleAnimation Storyboard.TargetName="PressedOverlay"
+															 Storyboard.TargetProperty="Opacity"
+															 Duration="{StaticResource MaterialAnimationDuration}"
+															 To="1">
+											<DoubleAnimation.EasingFunction>
+												<CubicEase EasingMode="EaseIn" />
+											</DoubleAnimation.EasingFunction>
+										</DoubleAnimation>
+									</Storyboard>
+								</VisualState>
+
+								<VisualState x:Name="Pressed">
+									<!-- Workaround for Issue #246732 where IOS combobox sometimes keeps the pressed-on state too long. -->
```

Comment:
It would be best to add the Issue link directly in the comment no? (instead of the number to go to the correct repo)

Reply:
@agneszitte Thanks for the comment. This was a code I copied from a Sample App. Let me check if the issue this is still an issue, and if so, would you allow me to push a new PR to update this? That way, we could merge this PR that has been idle for a few weeks.

Reply:
Yes that works for me @ajpinedam :) !